core/controlplane/config/templates/cloud-config-controller

```diff
@@ -345,11 +347,16 @@ coreos:
       command: start
       content: |
         [Unit]
-        Wants=kubelet.service docker.service
-        After=kubelet.service
+        Wants=kubelet.service docker.service install-kube-system.service
+        After=kubelet.service install-kube-system.service
 
         [Service]
-        Type=oneshot
+        Type=simple
+        Restart=on-failure
+        RestartSec=60
```

Comment:
Indeed, cfn-signal is at the end of the services chain so has no units dependent on it.